### PR TITLE
Only show inner text on failure of toContainText matcher

### PR DESCRIPTION
**Pull request by rupurt**
This is for `1.7.x`. I can also submit another one for master if you would like.

The old output included the entire DOM structure which made it hard to parse

### CODE REVIEW

**Comment by travisjeffery** [comment]
Hey i'm good to merge this, do you mind updating your branch so there's a clean merge? Thanks!

**Comment by rupurt** [comment]
@travisjeffery it looks like this [may already be done on master](https://github.com/velesin/jasmine-jquery/blob/master/lib/jasmine-jquery.js#L501). We're still on the 1.7.x branch as we haven't upgraded to Jasmine 2 yet.
